lib/datasource/packagist/index.ts

```diff
@@ -199,10 +199,10 @@ async function packagistOrgLookup(name: string): Promise<ReleaseResult> {
   if (cachedResult) {
     return cachedResult;
   }
+
   let dep: ReleaseResult = null;
   const regUrl = 'https://packagist.org';
-  const pkgUrl = URL.resolve(regUrl, `/p/${name}.json`);
-  // TODO: fix types (#9610)
+  const pkgUrl = joinUrlParts(regUrl, `/p2/${name}.json`);
```

Comment:
```suggestion
  const pkgUrl = joinUrlParts(regUrl, `/p2/${name}.json`);
  // TODO: fix types (#9610)
```
Don't remove the comment, as you don't fixed it😉

Reply:
Removed by mistake 😅